docs/artifacts.md

```diff
@@ -155,6 +155,82 @@ spec:
 
 It is recommended to use [purl format](https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst) for artifacts uri as shown in the example. 
 
+### Output Artifacts in SLSA Provenance
+By default, Tekton Chains will consider all output artifacts as `byProducts` when generating in the [SLSA provenance](https://slsa.dev/spec/v1.0/provenance). In order to treat an artifact as a [subject](https://slsa.dev/spec/v1.0/provenance#schema) of the build, you must set a boolean field `"isBuildArtifact": true` for the output artifact.
+
+e.g.  
+```yaml
+apiVersion: tekton.dev/v1
+kind: TaskRun
+metadata:
+  generateName: step-artifacts-
+spec:
+  taskSpec:
+    description: |
+      A simple task that populates artifacts to TaskRun stepState
+    steps:
+      - name: artifacts-producer
+        image: bash:latest
+        script: |
+          cat > $(artifacts.path) << EOF
+          {
+            "outputs":[
+              {
+                "name":"image",
+                "isBuildArtifact": true,
```

Comment:
nit: I'd drop the `is` - I don't know if this adds anything, and sometimes autogenerated clients (particularly if you ever deal with protobuf) will add prefixes that leads to clunky method names like `hasIsBuildArtifact`/`getIsBuildArtifact`

```suggestion
                "buildArtifact": true,
```

Reply:
Agreed! Much better. I'll use `BuildOutput` instead like you suggested above.